packages/slice-machine/lib/env/npmApi.ts

```diff
@@ -0,0 +1,58 @@
+import npmFetch from "npm-registry-fetch";
```

Comment:
This file is only used by the server, so we can put it in the server folder :)

Reply:
it's also only used here `packages/slice-machine/lib/env/semver.ts` make more sense to keep it next to the file where it's used?